### PR TITLE
[SEM - 006] - Primary Button with Icon 

### PR DESCRIPTION
**Pull request by VictoriaCk55**

### CODE REVIEW

**Comment by VictoriaCk55** [comment]
storybook

File: src/ui/components/atoms/Primary-Button/PrimaryButton.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import styles from './PrimaryButton.module.scss';
+import type { IProps } from './types/IProps';
+import Add from '../../../../assets/icons/add.svg?raw';
+
+function PrimaryButton(props: IProps) {
+  const { children, variant, onClick, icon } = props;
+  const selectedIcon = icon ?? (children === 'Primary button' ? Add : Add);
+  return (
+    <button
+      className={`${styles['primary-button']} ${styles[`primary-button--${variant}`]}`}
```
Comment (Ditmar) [inline src/ui/components/atoms/Primary-Button/PrimaryButton.tsx:10]: Nice pr, really good job, only a few things, please use this approach https://github.com/Ditmar/backoffice-openscience/blob/master/src/ui/components/demo/calculator/atoms/display.tsx to create your variants of the components.

**Review by Ditmar — CHANGES_REQUESTED** [review]

File: src/ui/components/atoms/Primary-Button/PrimaryButton.tsx (hunk 1 above)
Comment (VictoriaCk55) [reply src/ui/components/atoms/Primary-Button/PrimaryButton.tsx]: image
ya lo corregi ing

Comment (VictoriaCk55) [reply src/ui/components/atoms/Primary-Button/PrimaryButton.tsx]: image

Comment (VictoriaCk55) [reply src/ui/components/atoms/Primary-Button/PrimaryButton.tsx]: I request a review of my PR:
https://ditmar.github.io/backoffice-openscience/branch-feature/SEM-006/?path=/story/ui-components-atoms-primary-button

**Pull request merged**